terraform/main.tf

```diff
@@ -0,0 +1,37 @@
+locals {
+  database_users = setunion(local.products,
+    toset([
+      "bitwarden",
+      "vault",
+    ])
+  )
+  products = toset([
+    "common-funding-application",
+    "first-year-hub",
+    "office-hours-queue",
+    "penn-clubs",
+    "penn-courses",
+    "platform",
+    "platform-dev",
+    "student-life"
+  ])
+  iam_service_accounts = setunion(local.products,
+    toset([
+      "team-sync",
+      "db-backup",
+    ])
+  )
+  platform_members = toset([
+    "armaan",
+    "peyton"
+  ])
```

Comment:
shorturl.at/gqC78

Reply:
?

Reply:
Oh I guess the short URL broke :( [here's where it went to](https://memegenerator.net/img/instances/67951619/put-me-in-coach-im-ready.jpg)